streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -17,378 +17,831 @@
 
 package org.apache.kafka.streams.kstream;
 
+import org.apache.kafka.clients.producer.internals.DefaultPartitioner;
 import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.streams.KafkaStreams;
 import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.StreamsConfig;
+import org.apache.kafka.streams.kstream.internals.WindowedSerializer;
+import org.apache.kafka.streams.kstream.internals.WindowedStreamPartitioner;
 import org.apache.kafka.streams.processor.StreamPartitioner;
-import org.apache.kafka.clients.producer.internals.DefaultPartitioner;
+import org.apache.kafka.streams.state.ReadOnlyKeyValueStore;
 
 /**
  * {@link KTable} is an abstraction of a <i>changelog stream</i> from a primary-keyed table.
- * Each record in this stream is an update on the primary-keyed table with the record key as the primary key.
+ * Each record in this changelog stream is an update on the primary-keyed table with the record key as the primary key.
  * <p>
  * A {@link KTable} is either defined from one or multiple Kafka topics that are consumed message by message or
```

Comment:
remove "or multiple"
Unless there is something i'm missing a `KTable` is created from exactly one topic unless it is the result of an aggregation